policyengine_us/parameters/gov/states/ma/tax/income/exemptions/interest/in_effect.yaml

```diff
@@ -0,0 +1,15 @@
+description: The Massachusetts interest exemption is in effect if this is true.
+
+values:
+  2002-01-01: true
+  2024-01-01: false
+
+metadata:
+  unit: bool
+  period: year
+  label: Massachusetts interest exemption in effect
+  reference:
+    - title: Form 1 2024 Massachusetts Resident Income Tax - Repeal of the Deduction of Interest from Massachusetts banks
+      href: https://www.mass.gov/doc/2024-form-1-instructions/download#page=3
+    - title: Repeal Deduction of Interest From Savings in Massachusetts Banks 1
+      href: https://budget.digital.mass.gov/govbudget/fy25/outside-section/section-44-repeal-deduction-of-interest-from-savings-in-massachusetts-banks-1/
```

Comment:
add statutory reference to https://malegislature.gov/Laws/GeneralLaws/PartI/TitleIX/Chapter62/Section3 as well, with a comment indicating the relevant section has been deleted